.vscode/launch.json

```diff
@@ -0,0 +1,83 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Test Plugin",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": [
+        "run-script",
+        "tdd",
+        "${input:pluginName}"
+      ],
+      "env": {
+        "PLUGINS": "${input:pluginName}"
+      },
+      "port": 9229,
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Test Current File",
+      "runtimeExecutable": "npm",
+      "runtimeArgs": [
+        "run-script",
+        "tdd",
+        "${file}"
+      ],
+      "port": 9229,
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
+    },
+  ],
+  "inputs": [
+    {
+      "id": "pluginName",
+      "description": "The plugin name",
+      "type": "pickString",
+      "options": [
```

Comment:
This array should be added to the plugin checklist to not forget adding new plugins here.